LEGO1/lego/legoomni/include/legovideomanager.h

```diff
@@ -34,9 +34,12 @@ class LegoVideoManager : public MxVideoManager {
 	MxResult Tickle() override;                                                                       // vtable+0x08
 	void Destroy() override;                                                                          // vtable+0x18
 	MxResult Create(MxVideoParam& p_videoParam, MxU32 p_frequencyMS, MxBool p_createThread) override; // vtable+0x2c
-	MxResult RealizePalette(MxPalette*) override;                                                     // vtable+0x30
-	void UpdateView(MxU32 p_x, MxU32 p_y, MxU32 p_width, MxU32 p_height) override;                    // vtable+0x34
-	virtual MxPresenter* GetPresenterAt(MxS32 p_x, MxS32 p_y);                                        // vtable+0x38
+
+	MxPresenter* GetPresenterByActionObjectName(char* p_char);
```

Comment:
Do you prefer to keep the virtual functions together, or do you try to match the order in the binaries? I am open for suggestions for putting this decleration somewhere else.

Reply:
We generally keep the virtual functions together in one block. The order of the functions in the binaries doesn't need to be reflected in the header declarations